src/main/java/com/project/videotecha/repository/ReservationRepository.java

```diff
@@ -0,0 +1,17 @@
+package com.project.videotecha.repository;
+
+import com.project.videotecha.model.Reservation;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+
+import java.util.Optional;
+
+public interface ReservationRepository extends JpaRepository<Reservation, Long> {
+    @Query("SELECT coalesce (sum(r.numberOfSeats), 0) " +
+            "FROM Reservation r " +
+            "WHERE r.canceled = False and r.user.id = :user_id and r.projection.id = :projection_id ")
+    Integer sumNumberOfSeatsByUserIdAndProjectionId(@Param("user_id") Long userId, @Param("projection_id") Long projectionId);
```

Comment:
You can use triple quotes (""") for a string block here. Also, use newlines between where conditions for easier readability.